config/logback-stdout.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2014 Brandon Arp
```

Comment:
Is this being copied from somewhere that this isn't 2018?

Reply:
Is there a reason the copyright is you personally instead of ISM?

Reply:
Yeah, it was essentially copied from logback.xml